test/functional/test_framework/util.py

```diff
@@ -208,9 +208,11 @@ def str_to_b64str(string):
 def satoshi_round(amount):
     return Decimal(amount).quantize(Decimal('0.00000001'), rounding=ROUND_DOWN)
 
-def wait_until(predicate, *, attempts=float('inf'), timeout=float('inf'), lock=None):
+def wait_until(predicate, *, attempts=float('inf'), timeout=float('inf'), lock=None, factor=1.0):
     if attempts == float('inf') and timeout == float('inf'):
-        timeout = 60
+        timeout = 60 * factor
+    else:
+        timeout = timeout * factor
```

Comment:
```suggestion
    timeout = timeout * factor
```

can remove the else branch?